packages/typo3-docs-theme/assets/sass/_text.scss

```diff
@@ -1,7 +1,25 @@
-
+a {
+    color:inherit;
+    text-underline-offset: 25%;
+    &:hover {
+        text-decoration: none;
+    }
+}
 article {
     a:not([class*="btn"]) {
         text-decoration: underline;
+	    color: $secondary;
+        &:hover {
+            text-decoration: none;
+        }
+    }
+    p, li {
```

Comment:
The font size is not adjusted at all places:

before:
![Bildschirmfoto vom 2024-03-04 18-36-12](https://github.com/TYPO3-Documentation/render-guides/assets/2566282/e168e62b-49f6-4e9e-91bb-d2bd27963751)

after:
![Bildschirmfoto vom 2024-03-04 18-36-58](https://github.com/TYPO3-Documentation/render-guides/assets/2566282/2467d6bf-a313-40bd-84da-93fbf01994a0)

Reply:
I think, if we really change the font size we should do this at a more central place.

Additionally, for me the different font sizes looks strange:
- The font size of the content is larger
- Navigation stays smaller

Reply:
Maybe we should also largen the content width?

Reply:
Should we stick with one font size maybe?

Reply:
Indeed the font size should be the same in the central column, but it doesn't need to be the same in the navigation part like the aside menu or the breadcrumb. There is no accessibility rule saying that the font-size must be unique in an interface.

Reply:
done

Reply:
I am not speaking about accessibility. I am speaking that this looks not good to me, "unround" would I say in German.